### PR TITLE
update footer with UCSF badge

### LINKED ISSUE #11: Add branding to footer
we should add the UCSF branding to the site, as well as the "medbiquitous member" banner.

Collateral has been placed in the media dir.

### PR DESCRIPTION
**Pull request by saschaben**
fixes #11 
 - [x] add UCSF logo to footer
 - [x] add Medbiquitous member badge

### CODE REVIEW

**Comment by saschaben** [comment]
@trott I tried to do it with a div, and failed. If someone more clever can replace the nonbreaking space hackery with a subclass that will get the layout handled, that's dandy. But otherwise....

File: _includes/footer.html [hunk 1]
```diff
@@ -47,12 +47,13 @@
           {% endif %}
         </ul>
       </div>
-
+
```
Comment (Trott) [inline _includes/footer.html]: superfluous whitespace chars

**Comment by Trott** [comment]
That `&nbsp;` stuff makes you a terrible person. I'll come up with something else. Do not merge this yet,

File: _includes/footer.html [hunk 2]
```diff
@@ -47,12 +47,13 @@
           {% endif %}
         </ul>
       </div>
-
+      
       <div class="footer-col  footer-col-3">
-        <p class="text">{{ site.description }}</p>
+              <span class="text">a product of</span> <img src="/media/UCSF_SOM_logo.png" width="163" border="0">
```
Comment (Trott) [inline _includes/footer.html]: Do not scale PNGs down with `width` attributes. Resize the actual PNGs.

File: _includes/footer.html [hunk 3]
```diff
@@ -47,12 +47,13 @@
...
+              <p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
+              <img src="/media/MEDBIQ-logo.png" width="81" border="0"></p>
```
Comment (Trott) [inline _includes/footer.html]: Same as above re: PNGs and scaling.

File: _includes/footer.html (hunk 2 above)
Comment (Trott) [inline _includes/footer.html]: This image needs an `alt` attribute.

File: _includes/footer.html (hunk 3 above)
Comment (Trott) [inline _includes/footer.html]: This image needs an `alt` attribute too.

**Comment by Trott** [comment]
OK, `&nbsp;` and a bunch of other crimes against our users have been removed. Please test on different browsers and viewports.

**Pull request merged**